tsl/src/continuous_aggs/invalidation.c

```diff
@@ -514,16 +558,18 @@ cagg_invalidations_scan_by_hypertable_init(ScanIterator *iterator, int32 cagg_hy
 
 static void
 cut_cagg_invalidation(const CaggInvalidationState *state, const InternalTimeRange *refresh_window,
-					  Invalidation *entry)
+					  const Invalidation *entry, Invalidation *remainder)
```

Comment:
Nit: Since you're making changes here, I would suggest to move the object to be first in the parameter list.

Reply:
Again, remainder is where the result is returned.